specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -4026,10 +4026,34 @@
         "forwardDeadLetteredMessagesTo": {
           "type": "string",
           "description": "Queue/Topic name to forward the Dead Letter message"
+        },
+        "clientAffine": {
```

Comment:
>clientAffine [](start = 9, length = 12)

Do you think `isClientAffine` might be better for a boolean?

Reply:
Similarly, for `isDurable`, `isShared`

---
In reply to: [416216417](https://github.com/Azure/azure-rest-api-specs/pull/9063#discussion_r416216417) [](ancestors = 416216417)